core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -403,13 +437,17 @@ public Tenant getTenant(int tenantId) throws UserStoreException {
 
             result = prepStmt.executeQuery();
             boolean tenantUUIDColumnExists = hasColumn(result, COLUMN_NAME_UM_TENANT_UUID);
+            boolean associatedOrgUUIDColumnExists = hasColumn(result, COLUMN_NAME_UM_ORG_UUID);
 
             if (result.next()) {
                 id = result.getInt(COLUMN_NAME_UM_ID);
                 String domain = result.getString(COLUMN_NAME_UM_DOMAIN_NAME);
                 String email = result.getString(COLUMN_NAME_UM_EMAIL);
                 boolean active = result.getBoolean(COLUMN_NAME_UM_ACTIVE);
-                String associatedOrgID = result.getString(COLUMN_NAME_UM_ORG_UUID);
+                String associatedOrgID = null;
+                if (associatedOrgUUIDColumnExists) {
+                    result.getString(COLUMN_NAME_UM_ORG_UUID);
```

Comment:
don't we need to set this value to `associatedOrgID` ?

Reply:
fixed in [c5d56a3](https://github.com/wso2/carbon-kernel/pull/3316/commits/c5d56a3a83eca6381d1c85f1f3f0cb515b490c76)

Reply:
resolved in [c5d56a3](https://github.com/wso2/carbon-kernel/pull/3316/commits/c5d56a3a83eca6381d1c85f1f3f0cb515b490c76)